WordPress/Classes/ViewRelated/Tools/SettingsListPickerViewController.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import WordPressShared
+
+
+
+/// SettingsListPicker will render a list of options, and will allow the user to select one from the list.
+///
+class SettingsListPickerViewController : UITableViewController
+{
+    /// Header Strings to be applied over the diferent sections
+    ///
+    var headers : [String]?
+
+    /// Footer Strings to be applied over the diferent sections
+    ///
+    var footers : [String]?
+    
+    /// Titles to be rendered
+    ///
+    var titles : [[String]]?
+
+    /// Row Subtitles. Should contain the exact same number as titles
+    ///
+    var subtitles : [[String]]?
+    
+    /// Row Values. Should contain the exact same number as titles
+    ///
+    var values : [[NSObject]]?
```

Comment:
My first choice was definitely `AnyObject`. Here's the deal: AnyObject doesn't warrantee the implementation of the comparison operator. Meaning that: you can't really determine which object should get the checkmark.

For that reason, i had to switch over to NSObject. If there's a lightweight alternative (that doesn't involve defining yet another protocol, or adding too much code), i'd be more than happy to hack it!

CC @astralbodies 

Thanks Sergio :smile: